packages/server/api/routes/orderItems.router.js

```diff
@@ -0,0 +1,63 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const orderItemsController = require('../controllers/orderItems.controller');
+
+router.put('/:orderID/:movieID', (req, res) => {
```

Comment:
I often like url format as /orderID/:orderID/movieID/:movieID

Reply:
@itsgauravsaxena  so the url will look like localhost://5001/api/**order-items/orderID**/1/**movieID**/1?

Reply:
Yeah - but its my personal opinion :)

Reply:
Should I make it same as delete request?

Reply:
Yeah that seems close except missing userID part
/userID/:userID/:movieID/:orderID